processor/src/test/java/org/mapstruct/ap/test/constructor/nestedsource/_target/AdderUsageObserver.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.constructor.nestedsource._target;
+
+/**
+ * @author Sjaak Derksen
```

Comment:
are you sure?

Reply:
Yes (although I need to delete this one it is not used). I copied a lot of classes that you created for other things so I kept the name 😄